src/js/popup.js

```diff
@@ -348,6 +360,48 @@ function deactivateOnSite() {
   });
 }
 
+/**
+ * Open the share overlay (TODO currently an alert)
+ */
+function share() {
+  $("#share_overlay").toggleClass('active');
```

Comment:
Should we wait to make the overlay visible until after the text is populated, or does it not matter?

Reply:
Doesn't seem to matter; possibly on exceptionally slow computers the text generation might take a few frames, but I don't think it's significant.